### PR TITLE
DOCS-895: Add note on simultaneous config edits

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/manage/configuration.md [hunk 1]
```diff
@@ -74,6 +74,12 @@ You can use the mode selector to switch between **Builder** and **Raw JSON**:
 
 Regardless of the mode you choose, Viam stores the configuration file in [JSON (JavaScript Object Notation)](https://en.wikipedia.org/wiki/JSON).
 
+{{< alert title="Simultaneous config edits" color="caution" >}}
+If you edit a config while someone else edits the same config, the person who saves last will overwrite any other changes.
```
Comment (sguequierre) [inline docs/manage/configuration.md:78]:
```suggestion
If you edit a config while someone else edits the same config, the person who saves last will overwrite any prior changes.
```
(opt. Maybe this clarification could be useful) 
/ past / any past / previous?

**Review by sguequierre — APPROVED** [review]
LGTM one suggestion

File: docs/manage/configuration.md (hunk 1 above)
Comment (npentrel) [reply docs/manage/configuration.md:78]:
```suggestion
If you edit a config while someone else edits the same config, the person who saves last will overwrite any prior changes that aren't reflected in the new config.
```

File: docs/manage/fleet/_index.md [hunk 2]
```diff
@@ -69,7 +69,14 @@ Robot [configuration](robots/#configuration) and robot [code](#control-with-the-
 
 ### Configuration
 
-Everyone who has access to the location the robot is in, can change the robot's configuration.
+Everyone who has access to the location the robot is in, can change the robot's [configuration](robots/#configuration).
+
+{{< alert title="Simultaneous config edits" color="caution" >}}
+If you edit a config while someone else edits the same config, the person who saves last will overwrite any other changes.
```
Comment (npentrel) [inline docs/manage/fleet/_index.md:75]:
```suggestion
If you edit a config while someone else edits the same config, the person who saves last will overwrite any prior changes that aren't reflected in the new config.
```

**Pull request merged**